README.md

```diff
@@ -16,35 +21,35 @@ echo 'eval "$(~/.rbenv/bin/rbenv init - bash)"' >> ~/.bash_profile
 echo 'eval "$(~/.rbenv/bin/rbenv init - zsh)"' >> ~/.zshrc
 ```
 
-2. The ruby version used in this project is defined in the file `.ruby-version`living in the root of the application.
-Before proceeding run `rbenv install {VERSION}` where VERSION is version defined in `.ruby-version` file
+2. The `.ruby-version` file within the project's root directory defines the required Ruby version. Before proceeding, execute the following command to install the specified version using rbenv:
+```bash
+rbenv install {VERSION}
+```
 
 3. [Bundler](https://bundler.io/)
-Bundler provides a consintent environment for Ruby projects by tracking and installing the exact gems and versions that are needed.
-Bundler is an exit from dependency hell, and ensures that the the gems you need are present in development, staging, and production. Starting work on a project is as simple as 
+Bundler safeguards a uniform development environment for Ruby projects by managing exact gem dependencies and versions. It eliminates dependency-related issues and ensures consistency across development, staging, and production environments. To initiate dependency installation, execute: 
 ```bash
 bundle install
 ```
 4. Artifactory credentials
```

Comment:
it's preferable if we can point this section to a guide on backbase.io, if there's no guide we can leave this as is

Reply:
The bundler steps are not captured in the [getting started guide](https://backbase.io/developers/documentation/mobile-devkit/getting-started/set-up-ios-development/) so for now we have them in the README.md then once we improve the getting started document we can move them there.